eraser-map.yaml

```diff
@@ -794,6 +794,16 @@ styles:
         base: points
         texture: pois
         interactive: true
+        
+    icons-overlay:
+        base: points
+        texture: pois
+        interactive: true
+        blend: overlay
```

Comment:
Geraldine modified this in: 
https://github.com/tangrams/eraser-map/commit/8ee756535027d24b056242429a42da34c27022e0

To remove the `icon-overlay` style and reference, and move the `lines-overlay` above the `icons` style so that icons were always on top of the route line. Which doesn't make a whole lot of sense, but works.